### PR TITLE
bugfix: calculate current config value or arg at runtime

### PR DESCRIPTION
**Pull request by chrox**
This patch fixed the screen rotation bug that once going into landscape mode from document menu the config menu cannot rotate screen any more.

### CODE REVIEW

**Comment by houqp** [comment]
Thanks for the fixes and Happy Chinese new year!

**Comment by chrox** [comment]
The postInitCallback facility is great. I will fix it that way tonight. And I think it's better to use screen dimension explicitly in page hash since it will save several CPU cycles.

**Comment by chrox** [comment]
Thanks. And happy Spring Festival!

File: frontend/ui/reader/readerview.lua [hunk 1]
```diff
@@ -132,12 +132,21 @@ end
...
 function ReaderView:onSetScreenMode(new_mode)
 	if new_mode == "landscape" or new_mode == "portrait" then
+		self.screen_mode = new_mode
 		Screen:setScreenMode(new_mode)
 		self.ui:handleEvent(Event:new("SetDimensions", Screen:getSize()))
 	end
 	return true
 end
 
+function ReaderView:onRestoreScreenMode(old_mode)
```
Comment (houqp) [inline frontend/ui/reader/readerview.lua:142]: This event handler is created just for restoring screen mode when returning to filechooser right?

**Pull request merged**